.rubocop.yml

```diff
@@ -24,19 +24,19 @@ Performance:
     - '**/test/**/*'
 
 # Prefer assert_not over assert !
-Rails/AssertNot:
+ActionView/AssertNot:
```

Comment:
It is not correct to change this name, these rules are defined in https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsassertnot with the `Rails` namespace.

Reply:
All good, I understand. Can workaround for Jets. Closing out. Thanks for the response though!

Reply:
Oops. Ok